style.css

```diff
@@ -1,42 +1,79 @@
-/*
- * Mums in Tech | Week 03
- *
- * I have put some basic CSS in place to give everyone a
- * nice starting point. Feel free to change it however.
- *
- * You challenge, if you choose to accept it, is to
- * put your own design/ideas into the provided HTML with
- * minimal changes to the HTML itself.
- */
-
 body {
-  /* I prefer sans-serif (without curls) so I'm changing it haha */
-  font-family: Helvetica, sans-serif;
+  font-family: "Comic Sans MS", cursive, sans-serif;
+  background-color: #a9c3a8;
+  background-image: url("http://sg.theasianparent.com/wp-content/blogs.dir/1/files/7-mothers-ruling-the-tech-world/we-can-do-it.jpg");
+  background-repeat: no-repeat;
+  background-position: left center;
+  background-attachment: fixed;
+  line-height: 1.3;
+  border-top: 5px solid #fff;
+}
+
+h1 {
+	text-align: center;
+	text-shadow: 1px 1px #16271a;
```

Comment:
The text shadow makes it a little bit more difficult to read. When I used text shadows I try and simulate real shadows often by using `rgba` allowing me to specify an alpha/transparency value.

```css
rgba(22, 39, 26, 0.3)
```